README.md

```diff
@@ -1,5 +1,203 @@
-# FCC Test Suite Template
+<a name="readme-top"></a>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  <!-- - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portolio] <a name="about-project"></a>
+
+This project entails a product landing page that displays different products , Their description and cost . The site also has a section where it shows some feautures of the company which includes fast shipping , premium materials and quality assurance.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<!-- 
+> Describe the tech stack and include only the relevant sections that apply to your project. -->
```

Comment:
Sometimes leaving comments on your code makes it messy

Reply:
- [ ] This is a nice comment but can you please elaborate the more on what you want the developer to do, perhaps to remove the commented code. It might be obvious to you and not so for others . 👍